frontend/src/libs/enums/app-route.enum.ts

```diff
@@ -9,6 +9,8 @@ const AppRoute = {
   CHATS_$ID: '/chats/:id',
   JOURNAL: '/journal',
   JOURNAL_$ID: '/journal/:id',
+  JOURNAL_$ID_QUERY: '/journal/:id?query=:query',
+  CHATS_$ID_QUERY: '/chats/:id?query=:query',
```

Comment:
It's not a good idea

Reply:
@v0ldemar01 without that we have this bug:

https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/6e45a440-42de-417c-b5bb-f8650746987d

Reply:
Got it, but it's not a good practise for us

Reply:
Try to figure out the another one

Reply:
@v0ldemar01 but we use link component to open chat/journal where i can pass only variables from app-route.enum.ts. so i should change it or may be leave?

Reply:
@v0ldemar01 i find only two variant: leave it and add to router-provider/libs/maps/maps.tsx Partial type, or add to link component additional type. I can`t find another variant without modifying url of the link in sidebar